.github/workflows/release.yaml

```diff
@@ -11,29 +11,27 @@ on:
 env:
   # bleh
   MAINNET: ${{ github.ref == 'refs/heads/mainnet/main' || (startsWith(github.ref,'refs/tags/v') && ( contains(github.ref,'.0.') || contains(github.ref,'.2.') || contains(github.ref,'.4.') || contains(github.ref,'.6.') || contains(github.ref,'.8.') )) }}
-
+  GOLANG_CROSS_VERSION: v1.15.2
 jobs:
   goreleaser:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - run: git fetch --prune --unshallow
       - uses: actions/setup-go@v2
-
+      - run: make modvendor
       - name: release dry-run
-        uses: goreleaser/goreleaser-action@v1
-        with:
-          args: --skip-validate --skip-publish
-          version: v0.133.0
-
-      - if: startsWith(github.ref,'refs/tags/v') && github.repository == 'ovrclk/akash'
-        name: docker login
-        run: echo '${{secrets.DOCKERHUB_PASSWORD}}' | docker login --username '${{secrets.DOCKERHUB_USERNAME}}' --password-stdin
```

Comment:
are we still publishing images?

Reply:
yah, `goreleaser` in `goreng/golang-cross` is publishing it